src/utils/clamp/clamp.ts

```diff
@@ -0,0 +1,11 @@
+export function clamp(num: number, min: number, max: number): number {
```

Comment:
add jsDoc for this

/**
 * Clamps a value between a minimum and maximum range.
 * 
 * @param value - The value to be clamped.
 * @param min - The minimum value.
 * @param max - The maximum value.
 * @returns The clamped value.
 */
```suggestion
export function clamp(value: number, min: number, max: number): number {
```